### PR TITLE
CSV support

### PR DESCRIPTION
**Pull request by orta**
Adds an option for outputting as CSV.
Considering this does everything it is set out to do, I've moved it to `1.0.0` too. No need for super low releases when it fits its scope correctly.

### CODE REVIEW

**Comment by orta** [comment]
Assigning to you @Kapin

File: lib/pod/command/podfile_info.rb [hunk 1]
```diff
@@ -72,14 +76,18 @@ def pods_info_hash(pods, keys=[:name, :homepage, :summary, :license])
...
+        UI.puts "name,version,homepage,summary,license" if type == :csv
 
         pods.each do |pod|
-          if in_md
-            UI.puts "* [#{pod[:name]}](#{pod[:homepage]}) [#{pod[:license][:type]}] - #{pod[:summary]}"
+          case type
+          when :md
+            UI.puts "* [#{pod[:name]} - #{pod[:version]}](#{pod[:homepage]}) [#{pod[:license][:type]}] - #{pod[:summary]}"
+          when :csv
+            UI.puts "#{pod[:name]},  #{pod[:version]}, #{pod[:homepage]}, \"#{pod[:summary]}\", \"#{pod[:license][:type]}\""
```
Comment (segiddins) [inline lib/pod/command/podfile_info.rb]: Why the spaces?

Comment (orta) [inline lib/pod/command/podfile_info.rb]: legit point, that was me trying to read it better, but it's not legit CSV

**Comment by Kapin** [comment]
I'll take a proper look later tonight

File: lib/pod/command/podfile_info.rb [hunk 2]
```diff
@@ -15,14 +15,18 @@ class PodfileInfo < Command
       def self.options
         [
             ["--all", "Show information about all Pods with dependencies that are used in a project"],
-            ["--md", "Output information in Markdown format"]
+            ["--md", "Output information in Markdown format"],
+            ["--csv", "Output information in Markdown format"]
```
Comment (Kapin) [inline lib/pod/command/podfile_info.rb]: This should read in CSV format

**Comment by Kapin** [comment]
Just the one comment otherwise LGTM and I'll update the gem when I get home

**Comment by Kapin** [comment]
Ping @orta

**Comment by orta** [comment]
oops, sorry sure :D

**Comment by orta** [comment]
Done

**Comment by joshkalpin** [comment]
I'll update the gem tonight

**Pull request merged**